### PR TITLE
feat: snap web

### PR DESCRIPTION
**Pull request by BeroBurny**

### CODE REVIEW

File: packages/adapter/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    /* Projects */
+    // "incremental": true,                              /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
+    // "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
+    // "tsBuildInfoFile": "./.tsbuildinfo",              /* Specify the path to .tsbuildinfo incremental compilation file. */
+    // "disableSourceOfProjectReferenceRedirect": true,  /* Disable preferring source files instead of declaration files when referencing composite projects. */
+    // "disableSolutionSearching": true,                 /* Opt a project out of multi-project reference checking when editing. */
+    // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
+
+    /* Language and Environment */
+    "target": "es2016",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```
Comment (Lykhoyda) [inline packages/adapter/tsconfig.json:14]: Is there any issues with the never versions?

File: packages/example/src/App.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.logo {
+  float: left;
```
Comment (Lykhoyda) [inline packages/example/src/App.css:2]: Is it the default style? I think it's better to use flex for content aligment

File: packages/example/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+const { Header, Content, Footer } = Layout;
+
+export function App(): JSX.Element {
+  return (
+    <Layout className="layout">
+      <Header className="header" style={{ height: 67 }}>
+        <div className="logo" />
+        <NavigationMenu />
+      </Header>
+      <Content style={{ padding: '50px 50px', minHeight: 'calc(100vh - 67px - 64px)' }}>
```
Comment (Lykhoyda) [inline packages/example/src/App.tsx:15]: 'calc(100vh - 67px - 64px)'
67 and 64 it's probably a Header and Footer height, can we make it a constants to not create a confusion from where this numbers comes from

File: packages/example/src/App.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import './App.css';
+import { Outlet } from 'react-router-dom';
+import { NavigationMenu } from './components/NavigationMenu';
+
+const { Header, Content, Footer } = Layout;
+
+export function App(): JSX.Element {
+  return (
+    <Layout className="layout">
+      <Header className="header" style={{ height: 67 }}>
```
Comment (Lykhoyda) [inline packages/example/src/App.tsx:11]: I'm not aware of the framework choice but you decided to keep classes so please don't use style property

File: packages/adapter/tsconfig.json (hunk 1 above)
Comment (BeroBurny) [reply packages/adapter/tsconfig.json:14]: it is autogenerated by vite so no clue :man_shrugging:

File: packages/example/src/components/NavigationMenu/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export { NavigationMenu } from './NavigationMenu';
```
Comment (Lykhoyda) [inline packages/example/src/components/NavigationMenu/index.ts:1]: Not sure that is needed to create index file. The imports work great just with the component name. For the modern webpack it' even not necessary to mention the whole path if the folder and file has the same name. This practice with index file coming from the time when our bundler were less advance :)

File: packages/example/src/routes/root/Root.tsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import { Outlet } from 'react-router-dom';
+
+export function Root(): JSX.Element {
+  // const location = useLocation();
```
Comment (Lykhoyda) [inline packages/example/src/routes/root/Root.tsx:4]: to remove

File: packages/example/src/routes/root/Transfer.tsx [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import { TransferFunds } from '../../components/TransferFunds';
+
+export function Transfer(): JSX.Element {
+  return (
+    <>
```
Comment (Lykhoyda) [inline packages/example/src/routes/root/Transfer.tsx:5]: Fragment wrap is not required

File: packages/example/src/components/NavigationMenu/index.ts (hunk 5 above)
Comment (BeroBurny) [reply packages/example/src/components/NavigationMenu/index.ts:1]: maybe that work with CRA but unfortunately it is not the case with Vite (as I know)

**Pull request merged**